src/services/lomakkeet/perustelut/muut.js

```diff
@@ -243,7 +243,176 @@ export const getVankilaopetusPerustelulomake = () => {
     }
   ];
 };
+export const getTyovoimakoulutuslomake = () => {
+  const code = 4;
+  return [
+    {
```

Comment:
One way is to create labels like you have been doing here. Titles / labels can also be created by using code and title properties of a category.

```
categories = [{
code: 1,
title: "Example",
components: [...]
}]
```